README.md

```diff
@@ -98,7 +103,7 @@ We've currently only implemented one strategy for identifying the hostname of a
 
 Our strategy uses the name of the CAPI `Machine` as the hostname. To determine the Machine name the provider follows the owner chain from the `IPAddressClaim` via the infrastructure provider resources to the `Machine`. This is used by searching through the owner references up to a depth of five.
 
-To enable setting DNS entries, set the `spec.dnsZone` parameter on the `InfobloxIPPool` to your desired zone. The resulting DNS entries will then be `<machine name>.<dnsZone>`. The DNS view will be set to `default.<dnsZone>`.
+To enable setting DNS entries, set the `spec.dnsZone` parameter on the `InfobloxIPPool` to your desired zone. The resulting DNS entries will then be `<machine name>.<dnsZone>`. The DNS view will be set to the value of `dnsView` (or `defaultDnsView` if not set).
```

Comment:
Could you be more explicit about the behavior here please: it'll try `Pool.spec.dnsView`, then `Instance.spec.defaultDNSView` and then fall back to `default(.<network view>)` (the last part was actually inaccurate before).